docs/pages/guide/02-start.md

```diff
@@ -0,0 +1,409 @@
+---
+layout: guide
+group: guide
+title: Setup and Installation Guide
+---
+
+* TOC
+{:toc}
+
+
+<span class="c29">Yavin UI browser support</span>
+-------------------------------------------------
+
+<span class="c11">Yavin UI is an HTML-based browser application that is
+supported on the most recent three major versions of the following
+browsers:</span>
+
+<span class="c11"></span>
+
+-   <span class="c11">Chrome</span>
+-   <span class="c11">Firefox</span>
+-   <span class="c11">Safari</span>
+
+<span class="c29">Yavin installation guide</span>
+-------------------------------------------------
+
+The “<span class="c40">Installation Guide</span><span class="c11">” link
+takes you through the step-by-step guide in installing all the packages
+and files needed to have a full operational Yavin on your system.
+</span>
+
+<span class="c29">About data ingestion and data sources</span>
+--------------------------------------------------------------
+
+Before you can use Yavin to visualize and analyze data, you need to
+configure the Elide portion of Yavin to access the data you wish to
+report on. You do so by defining a <span class="c48">data
+source</span><span class="c11"> including the database connection
+configuration itself and its schema.</span>
+
+### <span class="c9">Data source dialects</span>
+
+<span class="c11">Elide analytic APIs generate SQL queries against your
+target database(s). Elide must be configured with a Dialect to correctly
+generate native SQL that matches the database grammar. Elide supports
+the following dialects by default:</span>
+
+
+|      Name                       |  Availiability                        |
+|---------------------------------|---------------------------------------|
+| <span class="c40">H2</span>     | <span class="c22">Now</span>          |
+| <span class="c40">Hive</span>   | <span class="c22">Now</span>          |
+| <span class="c40">Presto</span> | <span class="c22">Now</span>          |
+| <span class="c33">MySQL</span>  | <span class="c22">Now</span>          |
+{:.table}
+
+More information on dialects and how to use them can be found at: <span
+class="c13"><a href="https://www.google.com/url?q=https://elide.io/pages/guide/v5/04-analytics.html%23dialects&amp;sa=D&amp;ust=1606228594025000&amp;usg=AOvVaw3kCKuvbnXUCJCPe8SCcGF7" class="c14">https://elide.io/pages/guide/v5/04-analytics.html#dialects</a></span><span
+class="c11"> </span>
+
+<span class="c29">About data models</span>
+------------------------------------------
+
+A semantic model is a simplified representation of the data that resides
+in your database.  Semantic models consist of concepts like Tables,
+Dimensions, and Measures.  While the physical database schema (The
```

Comment:
I think we need to call out three things in this paragraph:
 - A semantic model is how data is presented to the user.  It is flat (no relationships), and simple (no formulas - just names).
 - A physical model is how data is arranged in your database.  It could be relational or involve complex SQL expressions.
 - Building a semantic model in yavin involves three things:
   1. Defining the tables, measures, and dimensions you want to expose to the end user (the semantic model).
   2. Adding metadata like descriptions, categories, and tags to the semantic model to help users understand it.
   3. Mapping the simple semantic model to the potentially more complex physical model.   Every concept (table, measure, dimension) in the semantic model can be assigned a native SQL fragment that is used to generate queries against the physical model.